src/util.py

```diff
@@ -35,13 +35,43 @@ def add_major_minor_ticks(ax, x_minor_per_major=3, y_minor_per_major=3, labelsiz
 
         y_minor_per_major (:class:`int`):
             Number of minor ticks between adjacent major ticks on y-axis
+
+        basex (:class:`int`):
+            If the xaxis scale is logarithmic, this is the base for the logarithm. Default is base 10.
+
+        basey (:class:`int`):
+            If the yaxis scale is logarithmic, this is the base for the logarithm. Default is base 10.
+
+        linthreshx (:class:`int`):
+            A kwarg passed to SymmetricalLogLocator if the xaxis scale is
```

Comment:
`kwarg` is usually used to refer to keyword type arguments like the ones here: https://book.pythontips.com/en/latest/args_and_kwargs.html . Is it intentional  to refer to `linthreshx` as a `kwarg`?

Reply:
I would say it's appropriate to refer to `linthreshx` as a `kwarg` because this value is passed into `tic.SymmetricalLogLocator` in line 65 as a `kwarg`. [This](https://matplotlib.org/api/scale_api.html#matplotlib.scale.SymmetricalLogScale) is the documentation I was referring to if it helps.

Reply:
I see your reasoning; however, what I meant is as follows (though, this is neither an issue nor a big deal but let us just note it down for accurate use of terminology):

`linthreshx` is nothing but a single numerical-value argument in the "scope" of your function; it is thus not a `**kwargs`. In other words, you are not handling it as a "variable-length" argument, which is the nature of `**kwargs`, throughout your function. 

Why the documentation you provided refers to `**kwargs` for several functions is because those functions handle their input arguments as variable-length arguments, i.e. **kwargs, in their own scope, but it does not imply that a single argument from within your function becomes `**kwargs` when you input it to one of those functions. 

By the way, just fyi: The documentation you provided includes a number of functions that handle `**kwargs`, but documentation is of `matplotlib.scale` namespace. However, `SymmetricalLogLocator` function you call at **Line 65** is from  `matplotlib.ticker` namespace, and it has a definition as follows:

`matplotlib.ticker.SymmetricalLogLocator(transform=None, subs=None, linthresh=None, base=None)`

which means it does not handle `linthresh` as a `**kwargs`.

Please let me know if I am missing some point.

Reply:
Oh okay thank you for clarifying. That makes a lot more sense. I taught myself Python and never realized exactly what the terminology means. I will change the comments in the code.

Reply:
Totally understandable. This `**kwargs` thing could make some confusion to anybody. Btw, you are doing great in Python.